visible_to/BUILD

```diff
@@ -0,0 +1,344 @@
+# Package groups defined for use in kubernetes visibility rules.
```

Comment:
@monopole is the expectation that we'll end up maintaining package lists like this?  I can't say that I'm fan of the notion of trying to sort out where and what to add.  What if we tried organizing the code into high level packages with a dependency ordering like we did for staging?  The rules are coarse, but they keep code localized to its proper place in the dependency tree.  During the transition, we moved packages to their new home to force the ratchet tighter.